db/schema.rb

```diff
@@ -16,18 +16,19 @@
 
 ActiveRecord::Schema.define(version: 20_210_808_180_355) do
   create_table 'cinema_halls', force: :cascade do |t|
-    t.string 'name'
-    t.integer 'capacity'
+    t.string 'name', null: false 
```

Comment:
Remember to always run and regenerate schema after adding migrations. Of course you don't have to change it here, but it's a generally good practice to do it and commit changes related to specific migration with schema. 

For instance, those changes - `nullability` was already provided in one of last PR's.
Also changes provided in migration from current PR - adding `seats` column are not visible here.

Generally, it's nothing bad in this phase of creating application, but when you will work with production code, that kind of mistake may give you a lot of problems with application deployment.